### PR TITLE
billing test

### PR DESCRIPTION
**Pull request by wanglin100**

### CODE REVIEW

File: modules/billing/src/test/java/org/eclipse/xpanse/modules/billing/providers/HuaweiBillingServiceTest.java [hunk 1]
```diff
@@ -4,70 +4,90 @@
...
+        DeployResourceEntity deployResourceEntity1 = new DeployResourceEntity();
+        deployResourceEntity1.setId(UUID.fromString("e97f9443-3850-40be-812b-c823fc652979"));
+        deployResourceEntity1.setKind(DeployResourceKind.VM);
+        deployResourceEntity1.setResourceId("d681840d-4f91-4148-b73d-ff2bc66786f5");
+        deployResourceEntity1.setName("basic-wmm");
+        SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
+        try {
+            deployResourceEntity1.setCreateTime(sdf.parse("2023-03-29 10:16:37.543"));
+        } catch (ParseException e) {
+            e.printStackTrace();
```
Comment (baixinsui) [inline modules/billing/src/test/java/org/eclipse/xpanse/modules/billing/providers/HuaweiBillingServiceTest.java]: We should use log.error() to replace e.printStackTrace();

File: modules/billing/src/test/java/org/eclipse/xpanse/modules/billing/providers/HuaweiBillingServiceTest.java [hunk 2]
```diff
@@ -4,70 +4,90 @@
...
+        deployResourceEntityList.add(deployResourceEntity1);
+        DeployResourceEntity deployResourceEntity2 = new DeployResourceEntity();
+        deployResourceEntity2.setId(UUID.fromString("4dbe2c13-5820-46c0-89d4-b3d5a6dfec9a"));
+        deployResourceEntity2.setResourceId("c6e38620-1d1c-4537-8f61-fb4f30b85fd1");
+        deployResourceEntity2.setName("volume");
+        try {
+            deployResourceEntity2.setCreateTime(sdf.parse("2023-03-29 10:16:37.543"));
+        } catch (ParseException e) {
+            e.printStackTrace();
+        }
```
Comment (baixinsui) [inline modules/billing/src/test/java/org/eclipse/xpanse/modules/billing/providers/HuaweiBillingServiceTest.java:78]: same above.

File: modules/billing/src/test/java/org/eclipse/xpanse/modules/billing/providers/HuaweiBillingServiceTest.java [hunk 3]
```diff
@@ -4,70 +4,90 @@
...
+        List<BillingDataResponse> billings = huaweiBillingService.onDemandBilling(service, true);
+        System.out.println(billings);
+        Assertions.assertFalse(CollectionUtils.isEmpty(billings));
 
     }
 
-
     @Test
     void getCsp() {
     }
```
Comment (baixinsui) [inline modules/billing/src/test/java/org/eclipse/xpanse/modules/billing/providers/HuaweiBillingServiceTest.java:98]: The getCsp() should not empty. use assertEquals(Csp.HUAWEI, billingService.getCsp());

File: modules/billing/src/test/java/org/eclipse/xpanse/modules/billing/providers/HuaweiBillingServiceTest.java (hunk 1 above)
Comment (wanglin100) [reply modules/billing/src/test/java/org/eclipse/xpanse/modules/billing/providers/HuaweiBillingServiceTest.java]: ok

File: modules/billing/src/test/java/org/eclipse/xpanse/modules/billing/providers/HuaweiBillingServiceTest.java (hunk 2 above)
Comment (wanglin100) [reply modules/billing/src/test/java/org/eclipse/xpanse/modules/billing/providers/HuaweiBillingServiceTest.java:78]: ok

File: modules/billing/src/test/java/org/eclipse/xpanse/modules/billing/providers/HuaweiBillingServiceTest.java (hunk 3 above)
Comment (wanglin100) [reply modules/billing/src/test/java/org/eclipse/xpanse/modules/billing/providers/HuaweiBillingServiceTest.java:98]: ok

**Pull request merged**